.octopus/variables.ocl

```diff
@@ -0,0 +1,23 @@
+variable "FeedApiKey" {
+
+    value "#{InternalFeedApiKey}" {
+        environment = ["components-internal"]
+    }
+
+    value "#{ExternalFeedApiKey}" {
+        description = ""
```

Comment:
We should clean these up when we migrate. I guess sometimes description gets set to an empty string.